resources/analysisTools/copyNumberEstimationWorkflow/impute2.sh

```diff
@@ -102,16 +102,16 @@ fi
 	
 				if [[ $var == 0 ]]
 				then
-					echo "WARNING: Non zero exit status during segmentation of segment ${SEGMENT} on chr ${CHR_NAME} in impute2.sh"
+					echo "WARNING: Non zero exit status during segmentation of segment ${SEGMENT} on chr ${CHR_NAME} in impute2.sh" >> /dev/stderr
```

Comment:
Why do we pipe this into a file? Doesn't this prevent us to find this in the final Roddy log file?